packages/firebase_auth/README.md

```diff
@@ -71,7 +71,7 @@ Future<FirebaseUser> _handleSignIn() async {
 }
 ```
 
-Then from the sign in button onPress, 
+Then from the sign in button onPress,
```

Comment:
Not your change, but that's a weird place to break this line.

Reply:
Done.